charts/seed-bootstrap/charts/elastic-kibana-curator/templates/searchguard/searchguard-deployment.yaml

```
@@ -0,0 +1,64 @@
+{{- if .Values.searchguard.enabled }}
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: sgadmin
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: logging
+    app: sgadmin
+    role: logging
+spec:
+  replicas: 1
```

```suggestion
  replicas: {{ .Values.searchguard.replicas }}
```
and you could pass `b.Shoot.GetReplicas(1)` to make it hibernation aware.